docs/docs/themes/using-multiple-themes.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Using multiple Gatsby themes.
+---
+
+Gatsby themes are intended to be composable. This means you can install multiple themes alongside each other.
+
+The `gatsby-starter-theme` composes two Gatsby themes: `gatsby-theme-blog` and `gatsby-theme-notes`
+
+```shell
+gatsby new my-blog https://github.com/gatsbyjs/gatsby-starter-theme
+```
+
+The starter includes both theme packages (`gatsby-theme-blog` and `gatsby-theme-notes`) in the starter's `gatsby-config.js` file.
+
+```javascript:title=gatsby-config.js
+module.exports = {
+  plugins: [
+    {
+      resolve: `gatsby-theme-notes`,
+      options: {
+        mdx: true,
+        basePath: `/notes`,
+      },
+    },
+    // with gatsby-theme-ui, the last theme in the config
+    // will override the theme-ui context from other themes
+    { resolve: `gatsby-theme-blog` },
+  ],
+  siteMetadata: {
+    title: `My site title`,
+  },
+}
+```
+
+In the default setup, a blog will be served from the root path (`/`), and the notes content will be served from `/notes`.
```

Comment:
Is there any more guidance we can add to talk about working with composed themes? Any gotchas, or related content?